project/Build.scala

```diff
@@ -105,7 +105,12 @@ object DottyBuild extends Build {
       (dottyOrganization % "dotty-sbt-bridge" % scalaVersion.value % "component").sources(),
 
     // sbt gets very unhappy if two projects use the same target
-    target := baseDirectory.value / ".." / "out" / name.value
+    target := baseDirectory.value / ".." / "out" / name.value,
+
+    // The non-bootstrapped dotty-library is not necessary when bootstrapping dotty
+    autoScalaLibrary := false,
```

Comment:
It's a bit more devious than that : we publish a dummy scala-library project (defined in Build.scala) that depends on dotty-library